src/arkit.cs

```diff
@@ -1395,6 +1590,381 @@ interface ARWorldMap : NSCopying, NSSecureCoding {
+	}
+
+	[iOS (13,0)]
+	[BaseType (typeof(NSObject))]
+	[DisableDefaultCtor]
+	interface ARQuickLookPreviewItem : QLPreviewItem {
+
+		[Export ("initWithFileAtURL:")]
+		[DesignatedInitializer]
+		IntPtr Constructor (NSUrl url);
+
+		[NullAllowed, Export ("canonicalWebPageURL", ArgumentSemantic.Strong)]
+		NSUrl CanonicalWebPageUrl { get; set; }
+
+		[Export ("allowsContentScaling")]
+		bool AllowsContentScaling { get; set; }
+	}
+
+	[iOS (13,0)]
+	[BaseType (typeof(NSObject))]
+	[DisableDefaultCtor]
+	interface ARRaycastQuery {
+
+		[Export ("initWithOrigin:direction:allowingTarget:alignment:")]
+		[MarshalDirective (NativePrefix = "xamarin_simd__", Library = "__Internal")]
+		IntPtr Constructor (Vector3 origin, Vector3 direction, ARRaycastTarget target, ARRaycastTargetAlignment alignment);
+
+		[Export ("origin")]
+		Vector3 Origin {
+			[MarshalDirective (NativePrefix = "xamarin_simd__", Library = "__Internal")]
+			get;
+		}
+
+		[Export ("direction")]
+		Vector3 Direction {
+			[MarshalDirective (NativePrefix = "xamarin_simd__", Library = "__Internal")]
+			get;
+		}
+
+		[Export ("target")]
+		ARRaycastTarget Target { get; }
+
+		[Export ("targetAlignment")]
+		ARRaycastTargetAlignment TargetAlignment { get; }
+	}
+
+	[iOS (13,0)]
+	[BaseType (typeof(NSObject))]
+	[DisableDefaultCtor]
+	interface ARRaycastResult {
+
+		[Export ("worldTransform")]
+		Matrix4 WorldTransform {
+			[MarshalDirective (NativePrefix = "xamarin_simd__", Library = "__Internal")]
+			get;
+		}
+
+		[Export ("target", ArgumentSemantic.Assign)]
+		ARRaycastTarget Target { get; }
+
+		[Export ("targetAlignment", ArgumentSemantic.Assign)]
+		ARRaycastTargetAlignment TargetAlignment { get; }
+
+		[NullAllowed, Export ("anchor", ArgumentSemantic.Strong)]
+		ARAnchor Anchor { get; }
+	}
+
+	interface IARSessionProviding {}
+
+	[Protocol]
+	interface ARSessionProviding {
+
+		[Abstract]
+		[Export ("session")]
+		ARSession Session { get; }
+	}
+
+	[iOS (13,0)]
+	[BaseType (typeof(NSObject))]
+	[DisableDefaultCtor]
+	interface ARSkeleton {
+
+		[Export ("definition")]
+		ARSkeletonDefinition Definition { get; }
+
+		[Export ("jointCount")]
+		nuint JointCount { get; }
+
+		[Export ("isJointTracked:")]
+		bool IsJointTracked (nint jointIndex);
+	}
+
+	[iOS (13,0)]
+	[BaseType (typeof(ARSkeleton))]
+	[DisableDefaultCtor]
+	interface ARSkeleton3D {
+
+		[EditorBrowsable (EditorBrowsableState.Advanced)]
+		[Protected, Export ("jointModelTransforms")]
+		IntPtr RawJointModelTransforms { get; }
+
+		[EditorBrowsable (EditorBrowsableState.Advanced)]
+		[Protected, Export ("jointLocalTransforms")]
+		IntPtr RawJointLocalTransforms { get; }
+
+		[Internal]
+		[Export ("modelTransformForJointName:")]
+		[MarshalDirective (NativePrefix = "xamarin_simd__", Library = "__Internal")]
+		Matrix4 _GetModelTransform (string jointName);
+
+		[Wrap ("_GetModelTransform (jointName.GetConstant ())")]
+		Matrix4 GetModelTransform (ARSkeletonJointName jointName);
```

Comment:
you can use `[BindAs]` but, if not, expose the other one as `NSString` (if only to avoid an unneeded cast / allocation)